src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -19,7 +19,7 @@ namespace Dynamo.Nodes
 {
     [NodeName("String")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("Creates a string.")]
+    [NodeDescription("StringInputNodeDescription", typeof(Properties.Resources))]
```

Comment:
Just found this when testing search tags, this seems to point to the wrong resource file.  Possibly also DoubleInput Below